jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -133,6 +133,24 @@ def test_sort(self, harness: primitive_harness.Harness):
       raise unittest.SkipTest("GPU tests are running TF on CPU")
     self.ConvertAndCompare(harness.dyn_fun, *harness.dyn_args_maker(self.rng()))
 
+  @primitive_harness.parameterized(primitive_harness.lax_linalg_qr)
+  def test_qr(self, harness: primitive_harness.Harness):
+    # See jax.lib.lapack.geqrf for the list of compatible types
+    if not harness.params["dtype"] in [jnp.float32, jnp.float64, jnp.complex64, jnp.complex128]:
+      with self.assertRaisesRegex(NotImplementedError, "Unsupported dtype"):
```

Comment:
It seems that on GPU the error raised here is `ValueError`.

Reply:
Addressed in latest commits